mam/pb3/pb3.h

```diff
@@ -29,58 +30,308 @@
 extern "C" {
 #endif
 
-trits_t pb3_trits_take(trits_t *const b, size_t const n);
+#define pb3_sizeof_oneof pb3_sizeof_tryte
+#define pb3_sizeof_repeated pb3_sizeof_size_t
 
 /**
- * Protobuf3 primitive types
- * null, tryte, size_t, trytes, tryte [n]
+ * Advances trits buffer by 'n'
+ *
+ * @param buffer - The buffer [in, out]
```

Comment:
For this and all following, `[in]`, `[out]` and `[in, out]` should be with the `@param` to be detected by Doxygen.
Like `@param[in]`

Reply:
@tsvisabo You didn't address this comment ?